package.json

```
@@ -73,9 +73,9 @@
   "homepage": "https://github.com/Semantic-Org/Semantic-UI-React#readme",
   "dependencies": {
     "@babel/runtime": "^7.10.5",
+    "@fluentui/react-component-event-listener": "^0.51.0",
+    "@fluentui/react-component-ref": "^0.51.0",
```

```suggestion
    "@fluentui/react-component-event-listener": "~0.51.0",
    "@fluentui/react-component-ref": "~0.51.0",
```

These packages do not follow Semver, let's be more accurate 😺 Please also run `yarn` after, it will fix version requirements in `yarn.lock`